content/articles/in-page-filtered-search-with-vanilla-javascript/index.md

```diff
@@ -0,0 +1,229 @@
+### Introduction
+You should allow visitors to search through a lot of data on a website if you have a page to find what they're looking for.  In my definition of "search," I don't mean browsing through a database or even JSON data; I mean searching through the content of a single web page. Make it easier for our users to find what they want by giving them a search option that breaks down the website into different parts. If you want to learn more about In-Runner Filtered Hunt With Vanilla and how it works, keep reading!

```

Comment:
This doesn't look like a good intro. Rather than directly jumping to a point, you give a brief overview of what are you talking about, and what will be covered in this tutorial?